server/blindai_app/src/main.rs

```diff
@@ -74,11 +74,11 @@ impl untrusted_local_app_server::UntrustedLocalApp for State {
     }
 }
 
-fn fill_blank_and_print(content: &str, size: usize)
-{
+fn fill_blank_and_print(content: &str, size: usize) {
     let trail_char = "#";
     let trail: String = trail_char.repeat((size - 2 - content.len()) / 2);
-    let trail2: String = trail_char.repeat(((size - 2 - content.len()) as f32 / 2.0).ceil() as usize);
+    let trail2: String =
```

Comment:
Assuming perfect arithmetic (no precision loss due to floating point numbers) (size - 2 - content.len()) as f32 / 2.0).ceil() as usize is actually the same as (size - 1 - content.len())/2  

todo: replace with the integer only formula